BehatCommands.php

```diff
@@ -0,0 +1,202 @@
+<?php
+namespace Drush\Commands\example_site_wide_command;
```

Comment:
This should probably be updated to `BehatDrushEndpoint` or similar?

Reply:
It will be `behat_drush_endpoint`